### PR TITLE
Feature/system reports

### PR DESCRIPTION
**Pull request by rbacordio**
- Fixed Duplicate Question on QuizSession Reports

### CODE REVIEW

File: WebApp/frontend/quiz-master/components/Commons/modals/ViewQuizSessionModal.tsx [hunk 1]
```diff
@@ -35,12 +35,16 @@ export default function ViewQuizSessionModal({
...
     useEffect(() => {
         const fetchQuestionInfos = () => {
+            const infos: Question[] = [];
             questionIds.forEach((id) => {
                 fetch(QUIZMASTER_QUESTION_GET_QUESTION + `${id}`)
                     .then((response) => response.json())
-                    .then((data: Question) =>
-                        setQuestionInfos((prev) => [...prev, data])
-                    )
+                    .then((data: Question) => {
```
Comment (jaymar921) [inline WebApp/frontend/quiz-master/components/Commons/modals/ViewQuizSessionModal.tsx:42]: looks good to me

File: WebApp/frontend/quiz-master/components/Commons/modals/ViewQuizSessionModal.tsx [hunk 2]
```diff
@@ -156,7 +160,7 @@ export default function ViewQuizSessionModal({
                                     >
                                         {({ loading }) =>
                                             loading
-                                                ? "Downloading document..."
+                                                ? "Preparing report"
```
Comment (jaymar921) [inline WebApp/frontend/quiz-master/components/Commons/modals/ViewQuizSessionModal.tsx:163]: nice

File: WebApp/frontend/quiz-master/app/reports/components/pdfReports/ParticipantAnswersTablePDF.tsx [hunk 3]
```diff
@@ -26,6 +26,10 @@ export function ParticipantAnswersTablePDF({
     participantAnswers: ParticipantAnswerReport[];
     questionInfos: Question[];
 }) {
+    console.log(
```
Comment (jaymar921) [inline WebApp/frontend/quiz-master/app/reports/components/pdfReports/ParticipantAnswersTablePDF.tsx:29]: might as well remove this

**Review by jaymar921 — APPROVED** [review]
All goods

**Review by kinloveko — APPROVED** [review]
Lgtmay

**Pull request merged**